frontend/pages/hosts/details/HostDetailsPage/HostActionsDropdown/helpers.tsx

```
@@ -317,6 +324,28 @@ const canShowRecoveryLockPassword = (config: IHostActionConfigOptions) => {
+    hostPlatform,
+    hostMdmEnrollmentStatus,
+    isManagedLocalAccountEnabled,
+  } = config;
+  if (!isPremiumTier) return false;
+  if (hostPlatform !== "darwin") return false;
+  if (!isConnectedToFleetMdm) return false;
+  if (!isAutomaticDeviceEnrollment(hostMdmEnrollmentStatus)) return false;
+  if (!isManagedLocalAccountEnabled) return false;
+  return (
+    isGlobalAdmin || isGlobalMaintainer || isTeamAdmin || isTeamMaintainer
+  );
```

ESLint

[error] Replace `(⏎····isGlobalAdmin·||·isGlobalMaintainer·||·isTeamAdmin·||·isTeamMaintainer⏎··)` with `isGlobalAdmin·||·isGlobalMaintainer·||·isTeamAdmin·||·isTeamMaintainer`

(prettier/prettier)